ansible_runner/utils.py

```diff
@@ -9,7 +8,7 @@
 import hashlib
 
 from collections import Iterable, Mapping
-
+from io import StringIO
```

Comment:
`six.StringIO` unless you're planning on dropping Python 2.7 support entirely.

Reply:
It looks like `io.StringIO` exists in 2.7 at least?